### PR TITLE
DEPM-584: Redirect triggers overview to new triggers overview

### PR DESCRIPTION
**Pull request by nathan-contino-mongo**

### CODE REVIEW

File: config/redirects [hunk 1]
```diff
@@ -12,3 +12,4 @@ raw: realm/authentication/linking -> ${base}/authentication
 raw: realm/getting-started/configure-rules-based-access-to-mongodb -> ${base}/mongodb/define-roles-and-permissions
 raw: realm/procedures/create-stitch-app -> ${base}/get-started/create-realm-app
 raw: realm/graphql/graphql-types-and-resolvers/index.html -> ${base}/graphql/types-and-resolvers/
+raw: realm/triggers/overview -> realm/triggers
```
Comment (jdestefano-mongo) [inline config/redirects:15]: Pretty sure this needs the `${base}/` prefix, which includes "realm" so we'll need to drop that too

```suggestion
raw: realm/triggers/overview -> ${base}/triggers
```

Comment (jdestefano-mongo) [reply config/redirects:15]: fwiw I don't know if the trailing slash matters or not. I posted the question in the platform channel, check there for an update.

**Review by jdestefano-mongo — CHANGES_REQUESTED** [review]
I believe there is an issue with the syntax and an open question with DOP. Once that's all sorted out, please deploy and test right away. Thanks!

File: config/redirects (hunk 1 above)
Comment (nathan-contino-mongo) [reply config/redirects:15]: d'oh. Thank you for noticing this!

Comment (nathan-contino-mongo) [reply config/redirects]: As for the trailing slash: my basic testing shows that either works in my browser. So perhaps it does not matter?

File: config/redirects [hunk 2]
```diff
@@ -12,3 +12,4 @@ raw: realm/authentication/linking -> ${base}/authentication
 raw: realm/getting-started/configure-rules-based-access-to-mongodb -> ${base}/mongodb/define-roles-and-permissions
 raw: realm/procedures/create-stitch-app -> ${base}/get-started/create-realm-app
 raw: realm/graphql/graphql-types-and-resolvers/index.html -> ${base}/graphql/types-and-resolvers/
+raw: realm/triggers/overview -> ${base}/triggers
```
Comment (nathan-contino-mongo) [inline config/redirects:15]:
```suggestion
raw: realm/triggers/overview -> ${base}/triggers/
```

**Review by nathan-contino-mongo — COMMENTED** [review]
adding trailing slash

**Review by jdestefano-mongo — APPROVED** [review]
lgtm

**Review by terakilobyte — APPROVED** [review]
LGTM

**Pull request merged**